packages/query/config/wallet-config.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
This seems to be more general settings. Does it belong inside of `query` ?

Reply:
+1 would not put the config here, but we could still have the query for it?